walkthroughs/week-3-server/examples/subtraction-game/src/main/webapp/index.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+!DOCTYPE html>
```

Comment:
I think you accidentally deleted  a <

Reply:
Thanks, I was wondering why this file kept on being marked as changed